libraries/Microsoft.Bot.Builder.Azure.CosmosDb/CosmosDbPartitionedStorage.cs

```diff
@@ -11,15 +11,15 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
-namespace Microsoft.Bot.Builder.Azure
+namespace Microsoft.Bot.Builder.Azure.CosmosDb 
 {
     /// <summary>
     /// Implements an CosmosDB based storage provider using partitioning for a bot.
     /// </summary>
     public class CosmosDbPartitionedStorage : IStorage, IDisposable
     {
         private const int MaxDepthAllowed = 127;
-        private readonly JsonSerializer _jsonSerializer = JsonSerializer.Create(new JsonSerializerSettings { TypeNameHandling = TypeNameHandling.All });
+        private readonly JsonSerializer _jsonSerializer = JsonSerializer.Create(new JsonSerializerSettings { TypeNameHandling = TypeNameHandling.None });
```

Comment:
Can we also update the recommendation in the public `CosmosDbPartitionedStorage(CosmosDbPartitionedStorageOptions cosmosDbStorageOptions, JsonSerializer jsonSerializer)` constructor down in this file? 
It may be good to note in the remarks section of that documentation that changing the serialization for deployed bots will break them

Reply:
I've changed this back to All in this PR. We can change the default in another PR, since this will also affect MemoryStorage and BlobsStorage defaults, as well as tests.

Also note: changing the default does not necessarily have to break existing bots, if we modify the dialog stack to generically specify types coming from storage. We can use some of the code from https://github.com/microsoft/botbuilder-dotnet/pull/4369